### ISSUE TITLE
I can not connect woocommerce, can you give me an example

### ISSUE THREAD

**Issue by marcelowu** [scottopolis/wpIonic#35]
I can not connect woocommerce, can you give me an example

**Comment by wubuyun** [comment]
https://yanz.com.br/wp-json/wc/v2/products?consumer_key=ck_898e264b385512f8153460a328a89d476b85da66&consumer_secret=cs_a0a02ef63770242c796c3d7d5c49a38e3b8294df
-->>OK...

**Comment by wubuyun** [comment]
'Basic Y2tfODk4ZTI2NGIzODU1MTJmODE1MzQ2MGEzMjhhODlkNDc2Yjg1ZGE2Njpjc19hMGEwMmVmNjM3NzAyNDJjNzk2YzNkN2Q1YzQ5YTM4ZTNiODI5NGRm';
--> does not...

**Comment by scottopolis** [comment]
This got moved to configure.ts. Download the newest version of this repo, then see the example in the readme: https://github.com/scottopolis/wpIonic

**Comment by rafasashi** [comment]
Hello,

I have the same Error here using Ionic 3.9.2 and the latest version of this repo.

The url with the key and secret works but not the 'Basic base64'.

I have also tried using btoa function but same.

I am testing from localhost (http) calling an https endpoint could it be the problem?  

How can I debug the http request and make sure the handler is sending the right headers?

I have also tried to start it with `ionic serve --ssl` but it is not working better.
